extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -3697,52 +3747,18 @@ var globalRequire = require;
 				 */
 				this.resumeAd = function(ad)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "ad". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)